build_tools/github/build_minimal_windows_image.sh

```diff
@@ -14,6 +14,10 @@ cp $WHEEL_PATH $WHEEL_NAME
 # Dot the Python version for identyfing the base Docker image
 PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1:2})
 
+# TODO: remove this when Python 3.12 is released
+if [[ "$PYTHON_VERSION" == "3.12" ]]; then
```

Comment:
Can we condition  on `CIBW_PRERELEASE_PYTHONS == True` here to and add the `"-rc"`?

When 3.12 gets released, we just have to remove `prerelease` from the build matrix, without needing to update this script anymore.